CollectNewspaperKarel.java

```diff
@@ -33,5 +39,11 @@ public void run(){
 		move();
 		turnAround();
```

Comment:
Don't you think this turnAround is extra for the method name? Method name is turnBackHome... which was completed before this turnAround? 

Although turnAround is needed but method name is saying that this turnAround step should not be part of this method... right?